networks.json

```diff
@@ -18,8 +18,8 @@
       "transactionHash": "0x4e20d94594f7ae340dadb18a9b7fa10e80f0a8dae2c72c27442cb015b9b205cf"
     },
     "4": {
-      "address": "0x4E608b7Da83f8E9213F554BDAA77C72e125529d0",
-      "transactionHash": "0xd1030e85508042a6762598935751d38c4acd537401e2f2980c8fba5261022691"
+      "address": "0x471B773019E5023DB37f33A774e5f4CC9C50CaBC",
+      "transactionHash": "0xfc3066fa7632f9ce8a0f258ce3d5886d9f2083c47ca27c17a5a0db466f419a53"
```

Comment:
This is a case of different versions deployed to different networks. In the end it's good that we didn't rely solely on deterministic addresses, otherwise we'd have to always redeploy on every network for every new version of the contract.